### PR TITLE
jbusecke added and build no bumped

### PR DESCRIPTION
**Pull request by jbusecke**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jbusecke** [comment]
I was unsure if I should bump the build no for this change. I will remove the bump if necessary

**Comment by ocefpaf** [comment]
> I was unsure if I should bump the build no for this change. I will remove the bump if necessary

In theory we don't need to bump b/c there is no functional change in the package,
but the package metadata is updated and we kind of want it to reflect the latest recipe.

TL;DR if you are about to about the recipe again don't bump so both can be done at one. If you are not planning on new changes anything soon bumping is OK.

**Comment by jbusecke** [comment]
I am pretty new to all of this, so ill try to familiarize myself with the process first. So no I am not planning on changing the version just now.

**Comment by ocefpaf** [comment]
No problem! Let's get this merged then :smile: 

Please take a look at https://github.com/conda-forge/staged-recipes#9-what-does-being-a-conda-forge-feedstock-maintainer-entail and let me know if you have any questions.
